### PR TITLE
Change "external registry" example in stack-yaml-reference

### PR DESCRIPTION
**Pull request by sanscontext**
### Describe the proposed changes

Gives a more generic registry example name.

### Please take a look

@borjaburgos @fermayo

### CODE REVIEW

File: docker-cloud/apps/stack-yaml-reference.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The image used to deploy this service. This is the only mandatory key.
 ```yml
 image: drupal
 image: dockercloud/hello-world
-image: quay.io/borja/redis
+image: my.registry.com/redis
```
Comment (johndmulhausen) [inline docker-cloud/apps/stack-yaml-reference.md]: This could just say `redis` as above; that's a library image, and the example will run that way.

Comment (sanscontext) [inline docker-cloud/apps/stack-yaml-reference.md]: This is specifically for an *external registry*; we're just moving away from using `quay.io` as a canonical "external registry".

Comment (borjaburgos) [inline docker-cloud/apps/stack-yaml-reference.md]: The biggest change here is that quay.io/borja/redis works and my.registry.com/redis doesn't.

Comment (sanscontext) [inline docker-cloud/apps/stack-yaml-reference.md]: @borjaburgos meaning that the external registry format is malformed? Or that `my.registry.com` is an example domain?

Comment (borjaburgos) [inline docker-cloud/apps/stack-yaml-reference.md]: Meaning that it is an non-working example domain. Otherwise, LGTM.

Comment (fermayo) [inline docker-cloud/apps/stack-yaml-reference.md]: It's an example domain. If we are worried about it not working with `my.registry.com/redis`, I'd remove it altogether.

Comment (sanscontext) [inline docker-cloud/apps/stack-yaml-reference.md]: Thanks for the clarification guys.  If anyone knows the owners of `example.com` and know how to get them to host a registry there, I'm all ears. ;)

**Comment by sanscontext** [comment]
Verbal LGTM from Borja and Fer, merging.

**Pull request merged**